packages/filter/src/reducer/func.ts

```diff
@@ -67,8 +67,14 @@ export const changeFilter = (
     }, ['drafts', `${index}`]);
   }
 
-  if (field === 'condition') {
-    const $state = setIn(state, value, ['drafts', `${index}`, 'condition']);
+  if (payload.field === 'condition') {
```

Comment:
minor: **field** value already extracted from payload at line 57.

Reply:
If  I use `field` ts don't see extensions key for this type
![image](https://user-images.githubusercontent.com/33321841/109849882-8ed29b80-7c5a-11eb-80fd-4bf85ae0a618.png)